### ISSUE TITLE
"region" role, aria-labelledby, and BaseAccordionPanel

### ISSUE THREAD

**Issue by charlie-yao** [charlie-yao/react-aria-widgets#194]
The default element for BaseAccordionPanel is `<section>`, and I thought it always had an implied role of `region`. Apparently, it only has the `region` role if it has an accessible label. Otherwise, it's `generic`. The implementation and documentation should be updated to account for this - e.g. should we provide a way to NOT set `aria-labelledby` if the developer wants to avoid landmark region proliferation?
